test/integration/features/cmd-image.feature

```diff
@@ -17,7 +17,7 @@ Feature: Basic image caching test
       And image export completes with 3 images within 20 minutes
       And container image "openshift\/origin-haproxy-router:v[0-9]+\.[0-9]+\.[0-9]+" is cached
       And container image "openshift\/origin-docker-registry:v[0-9]+\.[0-9]+\.[0-9]+" is cached
-      And container image "openshift\/origin:v[0-9]+\.[0-9]+\.[0-9]+" is cached
+      And container image "openshift\/origin-control-plane:v[0-9]+\.[0-9]+\.[0-9]+" is cached
```

Comment:
but we do not test here if OpenShift v3.9 got used...
In that case the control plane image should not be cached

Reply:
@gbraad we always use what default as per testing and soon when 3.11 release happen then 3.9 will be no longer support for minishift.